docs/sources/release-notes/v1-10.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Version 1.10 release notes
+menuTitle: V1.10
+description: Release notes for Grafana Pyroscope 1.10
+weight: 400
+---
+
+# Version 1.10 release notes
+
+The Pyroscope team is excited to present Grafana Pyroscope 1.10.
+
+This version of Pyroscope adds experimental support for OpenTelemetry profiles ([#2177](https://github.com/grafana/pyroscope/pull/2177)).
```

Comment:
Should this capability also be mentioned any place else in the docs? I'm inclined to wait until we have more examples and samples, since the team is working on that.

Reply:
yeah I'd say let's wait. cc @marcsanmi

Reply:
Yes, I agree 👍